Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/view/SettingsItem.java

```diff
@@ -22,6 +22,7 @@
 
   private String mKey;
   private String mSection;
+  private String gameId;
```

Comment:
Why not having this field inside the InputBindingSetting since it is the only use case for it?

Reply:
Good catch. Bindings "should" be the only settings this is needed for.